packages/eslint-plugin/tsconfig.json

```diff
@@ -1,6 +1,8 @@
 {
   "extends": "../../tsconfig.base.json",
   "compilerOptions": {
+    "module": "nodenext",
+    "moduleResolution": "nodenext",
```

Comment:
Not required for this PR I don't think, but should this be getting set for the entire repo via the base?

Reply:
Yeah, it's a big to-do; I thought this might help with the declaration emit errors, but it didn’t. This could be reverted but I'll leave it and follow up